delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -151,6 +157,20 @@ class ResourcesRoutesSpec extends BaseRouteSpec with IOFromMap {
       }
     }
 
+    "create a tagged resource" in {
+      val endpoints = List(
+        ("/v1/resources/myorg/myproject?tag=mytag", myId6, schemas.resources),
+        ("/v1/resources/myorg/myproject/myschema?tag=mytag", myId7, schema1.id)
+      )
+      forAll(endpoints) { case (endpoint, id, schema) =>
+        val payload = jsonContentOf("resources/resource.json", "id" -> id)
+        Post(endpoint, payload.toEntity) ~> routes ~> check {
+          status shouldEqual StatusCodes.Created
+          response.asJson shouldEqual resourceMetadata(projectRef, id, schema, (nxv + "Custom").toString)
+        }
+      }
+    }
```

Comment:
I think you need to look up the resource by the tag after creation. Otherwise the operation could do nothing and this test would pass

Reply:
@shinyhappydan Ah yeah, I checked that it was as expected in the route manually - but really a fake `Resources` should be injected in check that it's called with the right tag? But this is essentially an integration test so guess that isn't possible 😞 

Also could it make sense to have the tag on the creation response?

Anyway - updated with the check!

Reply:
I'm not for/against the tag being on the creation response, but it wouldn't help us determine that tag creation happened